exporter/clickhouselogsexporter/exporter.go

```diff
@@ -639,13 +639,13 @@ func (e *clickhouseLogsExporter) addAttrsToTagStatement(
 			e.logger.Debug("attribute value length exceeds the limit", zap.String("key", attrKey))
 			continue
 		}
+		e.addAttrsToAttributeKeysStatement(attributeKeysStmt, resourceKeysStmt, attrKey, tagType, utils.TagDataTypeString)
```

Comment:
Let's move this above the value length check on 638 line?

Reply:
But do we want to pollute our resource table as well ? there might be some wrong pipeline applied and because of which it will pollute the keys table as well.

Reply:
I didn't mean to allow the random keys. On 638, we are checking if the length of value >= 256 and continue. I can see a case where users have some keys with such long strings and wants to search on them. I think we should have such keys, even if the value is long. However, we don't want to have auto-complete on them which is reasonable.